torchio/transforms/augmentation/composition.py

```diff
@@ -65,12 +63,9 @@ def inverse(self, warn: bool = True) -> Transform:
                 message = f'Skipping {transform.name} as it is not invertible'
                 warnings.warn(message, RuntimeWarning)
         transforms.reverse()
-        if transforms:
-            result = Compose(transforms)
-        else:  # return noop if no invertible transforms are found
-            def result(x): return x  # noqa: E704
-            if warn:
-                warnings.warn('No invertible transforms found', RuntimeWarning)
+        result = Compose(transforms)
+        if not transforms:
```

Comment:
What about `if not transforms and warn`?

Reply:
Whoops! Good catch.

Just pushed a fix.